src/utils/handle-template-flags.js

```diff
@@ -0,0 +1,11 @@
+const getBaseOptions = require('./get-base-options');
+
+const handleFlags = async (flags) => {
+  if (flags === undefined || flags.noBaseData === false) {
+    return getBaseOptions();
+  }
+  console.warn('get base options has been disabled, make sure to get a project name in your template through prompts.');
+  return '';
```

Comment:
the returned type is string vs a Promise with undefined from `getBaseOptions`, I would either return `Promise.resolve()` or just `null`. Returning `Promise.resolve()` would be ideal since it give you context (tells you that `getBaseOptions` returns a promise without the need to open the file)